pages/common/docker-rename.md

```diff
@@ -0,0 +1,12 @@
+# docker rename
+
+> Rename a container.
+> More information: <https://docs.docker.com/engine/reference/commandline/rename>.
+
+- Show help:
+
+`docker rename`
+
+- Rename a container:
+
+`docker rename {{container}} {{new_name}}`
```

Comment:
```suggestion
- Rename a container:

`docker rename {{container}} {{new_name}}`

- Display help:

`docker rename --help`
```

We place the help and version examples in the last part of the page.

I would suggest going with `--help` flag here as it would suggest aliases too. I will add `docker-rename` as an alias page for this.

Ref.

```zsh
➜  ~ sudo docker rename
Place your right index finger on the fingerprint reader
"docker rename" requires exactly 2 arguments.
See 'docker rename --help'.

Usage:  docker rename CONTAINER NEW_NAME

Rename a container
➜  ~ sudo docker rename --help

Usage:  docker rename CONTAINER NEW_NAME

Rename a container

Aliases:
  docker container rename, docker rename
```